macOS/DBPE2ETests/DBPEndToEndTests.swift

```diff
@@ -296,7 +301,7 @@ final class DBPEndToEndTests: XCTestCase {
                 return optOutJobs.first?.lastRunDate != nil
             }
         })
-        print("Stage 5.1 passed: We start running the opt out jobs (only for non-removed brokers)")
+        print("Stage 5 passed: We start running the opt out jobs (only for non-removed brokers)")
```

Comment:
The `5` and `5.1` seemed backwards to me, let me know if this was intentional.

Reply:
It was intentional, but I'm not sure a good way to express it. Stage 5 is finishing running the opt outs, so then we need a way to express "we did part of stage 5".